src/main/java/seedu/address/logic/commands/NoteUpdateCommand.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.SeplendidModel;
+import seedu.address.model.notes.Content;
+import seedu.address.model.notes.Note;
+import seedu.address.seplendidui.UiUtil;
+
+/**
+ * Updates the content a note to the NoteList.
+ */
+public class NoteUpdateCommand extends NoteCommand {
+
+    public static final String NOTE_UPDATE_MESSAGE_USAGE = COMMAND_WORD
+            + " update [index] [content]: Updates a note to the specified content.";
+    public static final String ACTION_WORD = "update";
+    public static final String MESSAGE_SUCCESS = "Note updated: %1$s";
+    private final Integer noteIndexToUpdate;
+    private final Content updateContent;
+
+    /**
+     * Creates a NoteUpdateCommand to update the specified {@code Note}
+     *
+     * @param noteIndexToUpdate The Index of Note to be updated into Storage.
+     * @param updateContent The updated content of the note
+     */
+    public NoteUpdateCommand(int noteIndexToUpdate, Content updateContent) {
+        super();
+        requireNonNull(noteIndexToUpdate);
+        this.noteIndexToUpdate = noteIndexToUpdate;
+        this.updateContent = updateContent;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        return null;
+    }
+
+    @Override
+    public CommandResult execute(SeplendidModel seplendidModel) throws CommandException {
+        requireNonNull(seplendidModel);
+        int noteListSize = seplendidModel.getNoteCatalogue().getNoteList().size();
+        if (noteListSize < this.noteIndexToUpdate) {
+            throw new CommandException("Note not found, please put a valid index.");
```

Comment:
I think it would be better to make a private static final String MESSAGE_NONEXISTENT_NOTE to standardise across our code!

Reply:
okay will do!